<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrainSystem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Trilho1">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>40</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 1px solid #000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TremLaranja">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: orange;
border: 1px solid transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TremVermelho">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red;
border: 1px solid transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Trilho2">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>139</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 1px solid #000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TremAzul">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue;
border: 1px solid transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Trilho3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>139</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 1px solid #000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TremPreto">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>326</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
border: 1px solid transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Trilho4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>238</y>
      <width>140</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 1px solid #000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sem√°foro 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semaforo 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semaforo 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="s1Value">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>S1 value</string>
    </property>
   </widget>
   <widget class="QLabel" name="s2Value">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>S2 value</string>
    </property>
   </widget>
   <widget class="QLabel" name="s3Value">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>S3 value</string>
    </property>
   </widget>
   <widget class="QLabel" name="tremInS1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TremInS1</string>
    </property>
   </widget>
   <widget class="QLabel" name="tremInS2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TremInS2</string>
    </property>
   </widget>
   <widget class="QLabel" name="tremInS3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TremInS3</string>
    </property>
   </widget>
   <zorder>Trilho1</zorder>
   <zorder>Trilho2</zorder>
   <zorder>Trilho3</zorder>
   <zorder>Trilho4</zorder>
   <zorder>TremAzul</zorder>
   <zorder>TremPreto</zorder>
   <zorder>TremVermelho</zorder>
   <zorder>TremLaranja</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>s1Value</zorder>
   <zorder>s2Value</zorder>
   <zorder>s3Value</zorder>
   <zorder>tremInS1</zorder>
   <zorder>tremInS2</zorder>
   <zorder>tremInS3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
